browser/js/common/directives/navbar/navbar.html

```diff
@@ -1,30 +1,78 @@
-<nav class="navbar navbar-static-top">
-    <div class="container">
-        <ul>
-            <li>
-                <a ng-click="goToSection('.problem')"> The Problem </a>
-            </li>
+<nav id='navbar' class='navbar navbar-static-top'>
+	<div class='container'>
+		<div class='dropdown'>
+			<div class='dropdown-title'>
+				What is Kinetic
+			</div>
+				<div class='dropdown-content'>
+					<div class='list-container'>
+						<a ng-click="goToSection('.problem', 'home')"> The Problem </a>
+						<a ng-click="goToSection('.solution', 'home')"> Our Solution </a>
+						<a ng-click="goToSection('.what-is', 'home')"> What We Do </a>
+						<a ng-click="goToSection('.about-us', 'home')"> About Us </a>
+						<a ng-click="goToSection('.sponsored', 'home')"> Key Issues </a>
+						<a ng-click="goToSection('.get-involved', 'home')"> Get Involved </a>
+					</div>
+				</div>
+		</div>
```

Comment:
ah. Ok. So that would work for something like the main page where all the links the navbar contain are on the same page and thus the state for any of the `li` is the same but for any of the other dropdown lists where the links don't share the same state I don't think that would work. Unless we would then assign each `a` tag a `data-state` value and class name of where it should scroll to.